src/index.js

```diff
@@ -1 +1,72 @@
-require('./style.css');
\ No newline at end of file
+import axios from 'axios';
+
+require('./style.css');
+
+const id = '8dVjufgq9jkhppBCKQ2F';
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games';
+
+const generalList = document.getElementById('generalList');
+const submitBtn = document.getElementById('submit');
+const refreshBtn = document.getElementById('refreshBtn');
+const nameInput = document.getElementById('nameInput');
+const scoreInput = document.getElementById('scoreInput');
+
+const newGame = {
+  user: '',
+  score: '',
+};
+
+const addGame = async (game) => {
+  try {
+    await axios.post(`${url}/${id}/scores`, game, {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    nameInput.value = '';
+    scoreInput.value = '';
+    return 0;
+  } catch (error) {
+    return error;
+  }
+};
+
+submitBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  const nameInputValue = nameInput.value;
+  const scoreInputValue = scoreInput.value;
+  newGame.user = nameInputValue;
+  newGame.score = scoreInputValue;
+  addGame(newGame);
+});
+
+let gameArray = [];
+
+const getAllBooks = (arr) => {
+  // console.log('h');
```

Comment:
- [ ] Dear @Fahim-Khan-P, as the previous code reviewer mentioned, Kindly remove commented lines of code as this makes your codebase cleaner.

Reply:
This issue is well resolved! ✔️